packages/content-handler/src/websites/twitter-handler.ts

```diff
@@ -274,7 +274,7 @@ const getTweetIds = async (
         }
 
         window.scrollBy(0, distance)
-        await waitFor(100)
+        await waitFor(500)
```

Comment:
:-(

Reply:
Yeah, I know...

The issue is 100ms is not enough to load some tweets with big content so I have to increase it a bit